packages/headless/src/composables/listbox/createListbox.ts

```diff
@@ -93,6 +103,7 @@ export const createListbox = createBuilder(
         case " ":
           event.preventDefault();
           if (options.activeOption.value != undefined) {
+            // TODO: don't call onSelect if the option is disabled
```

Comment:
I don't think we need this because disabled options can not be highlighted via keyboard at all

Reply:
currently they can be selected. I was able to toggle the disabled checkbox option with space and arrow keys